### PR TITLE
trace messages

### PR DESCRIPTION
**Pull request by dho937**

### CODE REVIEW

File: bluebutton-data-pipeline-fhir-load/src/test/resources/logback-test.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 	<!-- Configure the root logger to filter to 'info' and more severe, and 
 		send all events to 'STDOUT'. -->
-	<root level="info">
+	<root level="trace">
```
Comment (karlmdavis) [inline bluebutton-data-pipeline-fhir-load/src/test/resources/logback-test.xml]: Pretty sure this wasn't necessary. At least... I don't think I've had to add it, myself, anyways.

File: bluebutton-data-pipeline-fhir-load/src/test/resources/logback-test.xml [hunk 2]
```diff
@@ -12,7 +12,7 @@
 	<!-- At 'debug', this will ensure that request and response headers are logged. 
 		At 'trace', this will ensure that request and response bodies are logged. -->
 	<logger name="gov.hhs.cms.bluebutton.datapipeline.fhir.load.FhirLoader"
-		level="info" />
+		level="trace" />
```
Comment (karlmdavis) [inline bluebutton-data-pipeline-fhir-load/src/test/resources/logback-test.xml]: Do we want this to be there always, or just as needed? I've been turning it down when I needed it, and reverting it before I committed, but maybe that doesn't make sense?

**Comment by dho937** [comment]
@karlmdavis  I only moved up the "trace" change to see if the unit tests would show the FHIR resources in an XML format with the data to be loaded as the project was being built on GitHub.  I am helping Abdul finish up the sample data in the CCW-to-FHIR mapping document so I thought this would help.  I can't get the FhirLoadIT JUnits to run locally yet to load the data because I don't have the local server set up correctly.  Will switch it back to "info" bottom line.

**Comment by karlmdavis** [comment]
Whoops: of course that's why you did it. Don't mind me; I was just being dumb.

**Comment by dho937** [comment]
Don't need to merge these code changes.

**Pull request closed without merging**